addons/point_of_sale/static/src/app/store/combo_configurator_popup/combo_configurator_popup.js

```diff
@@ -0,0 +1,38 @@
+/** @odoo-module */
+import { AbstractAwaitablePopup } from "@point_of_sale/app/popup/abstract_awaitable_popup";
+import { useState } from "@odoo/owl";
+import { usePos } from "../pos_hook";
+import { GenericProductCard } from "@point_of_sale/app/screens/product_screen/product/product";
+
+export class ComboConfiguratorPopup extends AbstractAwaitablePopup {
+    static template = "point_of_sale.ComboConfiguratorPopup";
+    static components = { GenericProductCard };
+
+    setup() {
+        super.setup();
+        this.pos = usePos();
+        this.state = useState({
+            combo: Object.fromEntries(this.props.product.combo_ids.map((elem) => [elem, 0])),
+        });
+    }
+
+    getTotalPrice() {
```

Comment:
When computing the display price, you need to take into account the "iface_tax_included" from the config.